design/FY2025/NFP-IdealLoadsAirSystem-FuelEfficiency.md

```diff
@@ -0,0 +1,111 @@
+Heating and Cooling Fuel Efficiency for Ideal Loads Air System
+==============================================================
+
+**Bereket Nigusse**
+
+**Florida Solar Energy Center**
+
+ - First draft: March 5, 2025
+ - Modified Date: NA
+
+## Justification for New Feature ##
+
+Ideal loads air systems are increasingly used for early-stage design analysis and in a building energy code compliance method. 
+However, currently this object has no built-in method for converting the loads to energy consumption. 
+As a result, users have to post-process simulation results to convert the loads to energy. 
+
+**- This enhancement is intended to support loads conversion into energy consumption for ideal loads air system object **
+**- Also reduces the burden of simulation results post processing and increase the usability of the object **
+
+## E-mail and  Conference Call Conclusions ##
+
+NA
+
+## Overview ##
+
+(1) ZoneHVAC:IdealLoadsAirSystem reports zone loads but not the equivalent energy consumption.
+(2) This object needs a method for converting loads into energy consumptions. 
+(3) It doesn't have input fields for specifying fuel efficiency values.
+
+
+## Approach ##
+
+The following two alternative approaches have been proposed for consideration to achieve the intended goal:
+
+These two options will be discussed and the full NFP will be drafted after the initial discussion.
+
+** Option I: Adding EMS Actuators **
+Adds EMS actuator for heating and cooling fuel efficiencies to the ZoneHVAC:IdealLoadsAirSystem object. 
+
+ * - Requires adding two actuators: *
+ * - (1) EMS actuator “Heating Fuel Efficiency” for heating *
+ * - (2) EMS actuator “Cooling Fuel Efficiency” for cooling *
+
+These actuators pass the efficiency values and give access to the conversion method to calculate the energy consumption during runtime.
+
+** Option II: Adding New INput Fields **
+Add optional heating and cooling fuel efficiency input fields to the ZoneHVAC:IdealLoadsAirSystem object. 
+The cooling and heating efficiency values will be used to calculate the energy consumed from the zone's ideal loads.
+
+ * - Requires adding two new input fields: *
+ * - (1) New Input Field: “Heating Fuel Efficiency” for heating *
+ * - (2) New Input Field: “Cooling Fuel Efficiency” for cooling *
+
+For both options, the heating and cooling energy rate and energy consumption will be calculated from the corresponding loads and fuel efficiencies as follows:
+
+	 {E_dot_{heat}} = {Q_dot_{load,heat}} / {Fuel_Eff_{heat}}
+	 {E_dot_{cool}} = {Q_dot_{load,cool}} / {Fuel_Eff_{cool}}
+	 
+	 where,
+	 {Q_{dot,cool}} 	= Zone Ideal Loads Cooling Rate, [W]
+	 {Q_{dot,heat}} 	= Zone Ideal Loads Heating Rate, [W]
+	 {E_dot_{cool}} 	= Zone Ideal Loads Cooling Rate, [W]
+	 {E_dot_{heat}} 	= Zone Ideal Loads Heating Rate, [W]
+	 {Fuel_Eff_{cool}} 	= Cooling Fuel Efficiency, [-]
+	 {Fuel_Eff_{heat}} 	= Heating Fuel Efficiency, [-]
```

Comment:
If you made these fields schedules, then you could have both inputs and the ability to use EMS actuators.

Reply:
@shorowit Good point. Your idea is to use the existing schedule actuators if needed to override the efficiency values in the schedule specified.

Reply:
Now, implemented using a schedule object for both cooling and heating fuel efficiency input fields.